make develop doesn't work

$ make develop
make: *** No rule to make target `develop'.  Stop.

Also "make dev" fails:

$ make dev
virtualenv env
New python executable in env/bin/python
Installing Setuptools..............................................................................................................................................................................................................................done.
Installing Pip.....................................................................................................................................................................................................................................................................................................................................done.
mkdir packages; \
	env/bin/pip install --download packages -r requirements/install.txt && \
	touch packages/.done
Could not open requirements file: [Errno 2] No such file or directory: 'requirements/install.txt'
Storing complete log in ~/.pip/pip.log

--- a/Makefile
+++ b/Makefile
@@ -17,20 +17,20 @@ env:
 deps: env packages/.done
 	# see http://tartley.com/?p=1423&cpage=1
 	# --upgrade needed to force local (if there's a system install)
-	$(PIP) install --no-index --find-links=file://$${PWD}/packages -r requirements/install.txt
+	$(PIP) install --no-index --find-links=file://$${PWD}/packages
 
 packages/.done:
 	mkdir packages; \
-	$(PIP) install --download packages -r requirements/install.txt && \
+	$(PIP) install --download packages  && \
 	touch packages/.done
 
 testdeps: deps packages/.test.done
-	$(TEST-PIP) install --no-index --find-links=file://$${PWD}/packages -r requirements/install.txt ;\
-	$(TEST-PIP) install --no-index --find-links=file://$${PWD}/packages -r requirements/testing.txt
+	$(TEST-PIP) install --no-index --find-links=file://$${PWD}/packages  ;\
+	$(TEST-PIP) install --no-index --find-links=file://$${PWD}/packages
 
 packages/.test.done:
 	mkdir packages; \
-	$(PIP) install --download packages -r requirements/testing.txt && \
+	$(PIP) install --download packages && \
 	touch packages/.test.done
 
 

--- a/README.rst
+++ b/README.rst
@@ -17,9 +17,9 @@ Install
 
 running::
 
-  make develop
+  make dev
 
-will create a virtualenv in ``env``. The ``rst2epb.py`` binary will be
+will create a virtualenv in ``env``. The ``rst2epub.py`` binary will be
 located in the ``env/bin/`` directory.
 
 Known to work on linux systems. (Should work on apple, cygwin, MS with

